doc/crud.md

```diff
@@ -410,108 +396,101 @@ For the second set of examples the original document is set up with a body and
 
 1. Update the attachments without changing the body, remove all attachments
    by setting an empty dictionary.
     ```objc
-    CDTMutableDocumentRevision *update = [saved mutableCopy];
-    update.attachments = @{};
+    saved.attachments = [@{} mutableCopy];
 
-    CDTDocumentRevision *updated = [datastore updateDocumentFromRevision:update
+    CDTDocumentRevision *updated = [datastore updateDocumentFromRevision:saved
                                                                    error:&error];
     ```
 
 1. Copy an attachment from one document to another.
     ```objc
     // Create a revision with attachments
-    CDTMutableDocumentRevision *rev = [CDTMutableDocumentRevision revision];
-    rev.docId = @"doc1";
-    rev.body = @{ ... };
+    CDTDocumentRevision *rev = [CDTDocumentRevision revisionWithDocId:@"doc1"];
+    rev.body = [@{ ... } mutableCopy];
     CDTUnsavedFileAttachment *att1 = /* blah */
-    rev.attachments = @{ att1.name: att1 };
+    rev.attachments = [@{ att1.name: att1 } mutableCopy];
     CDTDocumentRevision *revWithAttachments = [datastore createDocumentFromRevision:rev
                                                                               error:&error];
 
     // Add attachment to "saved" from "revWithAttachments"
-    CDTMutableDocumentRevision *update = [saved mutableCopy];
     CDTAttachment *savedAttachment = revWithAttachments.attachments[@"nameOfAttachment"];
-    update.attachments = @{savedAttachment.name: savedAttachment};
+    saved.attachments = @{savedAttachment.name: savedAttachment};
 
-    CDTDocumentRevision *updated = [datastore updateDocumentFromRevision:update
+    CDTDocumentRevision *updated = [datastore updateDocumentFromRevision:saved
                                                                    error:&error];
     ```
 
-### Creating a document from a `mutableCopy`
-
-It should be possible to create a new document from a `mutableCopy` of an existing document.
+### Creating a document by copying data
 
+It should be possible to create a new document by copying data from another
+document:
 
-1. Add a document from a `mutableCopy`, with attachments
+1. Copy a document with attachments
     ```objc
-    CDTMutableDocumentRevision *update = [saved mutableCopy];
-    update.docId = @"doc2";
-    update.body[@"hello"] = @"world";
+    CDTDocumentRevision *rev = [CDTDocumentRevision revisionWithDocId:@"doc2"];
+    rev.body = [saved.body mutableCopy];
+    rev.body[@"hello"] = @"world";
     // Create att100 attachment
-    update.attachments[att100.name] = att100;
+    rev.attachments = [saved.attachments mutableCopy];
+    rev.attachments[att100.name] = att100;
 
-    CDTDocumentRevision *updated = [datastore createDocumentFromRevision:update
+    CDTDocumentRevision *updated = [datastore createDocumentFromRevision:rev
                                                                    error:&error];
     ```
 
 1. Add a document from a `mutableCopy`, without attachments
```

Comment:
Has this example slightly changed from the heading? Now it's "copy a body from an existing document, without copying attachments" or something.